src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), colors should be defined in variables if they are used multiple times or are part of the color palette for the project. Consider adding this color to your variables if it meets the criteria.